openwisp_notifications/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import include, path
 
 from .api.urls import get_api_urls
+from .base.views import notification_preference_page
```

Comment:
This URL for this view should be added in openwisp_notifications/urls.py and not in the `api` sub-module. 

https://github.com/openwisp/openwisp-notifications/blob/63a29862e97faa1cc5847a149189be06171ee564/openwisp_notifications/urls.py#L6-L15